plugins/vi-mode/vi-mode.plugin.zsh

```diff
@@ -20,6 +20,9 @@ zle -N zle-keymap-select
 
 bindkey -v
 
+#rebind special key keybinds
+source $ZSH/lib/special-keys.zsh
```

Comment:
This doesn't respect `$ZSH_CUSTOM`, which might be trying to override `special-keys.zsh`.